### PR TITLE
Hide relocation information if policy has been rolled back

### PR DESCRIPTION
**Pull request by elceebee**
## Context
The international relocation payment is being rolled back, so we need to remove references to it in the services.

## Changes proposed in this pull request
- Remove international relocation payment information because the policy is being rolled back

## Guidance to review

In publish, I could only find references to international relocation payments in one view. Please have a look to see if you can find anything else. 

## Link to Trello card

https://trello.com/c/WvGQowjb

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by elceebee** [comment]
[Remove content that relates to the international relocation payment](https://trello.com/c/WvGQowjb/1463-remove-content-that-relates-to-the-international-relocation-payment)

**Review by dcyoung-dev — COMMENTED** [review]
Same comments as https://github.com/DFE-Digital/apply-for-teacher-training/pull/9230#pullrequestreview-1957185325

**Comment by elceebee** [comment]
[Remove content that relates to the international relocation payment](https://trello.com/c/WvGQowjb/1463-remove-content-that-relates-to-the-international-relocation-payment)

File: .gitignore [hunk 1]
```diff
@@ -47,6 +47,7 @@ terraform/aks/vendor/
 
 /.vscode/*
 /.devcontainer/*
+/.idea/*
```
Comment (dcyoung-dev) [inline .gitignore:50]: ❤️

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**